### PR TITLE
update plaid-python to match 2019-05-29 API release

### PR DESCRIPTION
**Pull request by tanatplaid**
This just fixes a bunch of tests and asserts a few more things.

We can update the version number too if desired.

### CODE REVIEW

File: tests/integration/util.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def create_client():
                   os.environ['SECRET'],
                   os.environ['PUBLIC_KEY'],
                   'sandbox',
-                  api_version="2017-03-08")
+                  api_version="2019-06-03")
```
Comment (ntindall) [inline tests/integration/util.py]: should be `2019-05-29`

**Comment by tanatplaid** [comment]
> We need to publish this as a major version, since the API has breaking changes.

the python library doesn't have any type/var assertions in it - it seems like a JSON.parse wrapper. i'm not sure we necessarily need to publish but I updated them anyway.

**Comment by r-ohan** [comment]
tanatplaid so the python library is the ~one~ library that doesn't need to get a major version bump on api version changes. all others have types / type definitions.

**Comment by tanatplaid** [comment]
as i expected. should i keep the version update change or nah?

**Comment by r-ohan** [comment]
yeah i don't think it makes much sense to bump the major version

**Comment by jasonjia93** [comment]
Seems fine not to bump the major version, but the patch or minor version should still be bumped

**Pull request merged**